CHANGELOG.md

```diff
@@ -1,26 +1,24 @@
 # Changelog
 
-All notable changes to Kubernetes and Helm resources for WSO2 API Management version `1.5.x` in each resource release,
+All notable changes to Kubernetes and Helm resources for WSO2 Open Banking version `2.0.0` in each resource release,
```

Comment:
Should be 2.0.x

Reply:
Created branch is 2.0.0 and previous branches were created in the format of 1.4/5.x. That's the reason.

Reply:
Let's go ahead with `2.0.x` because we will eventually create this branch as the development branch. `2.0.0` is only a temporary branch.

Reply:
Fixed with bfff48b.